src/main/java/com/crud/tasks/config/CoreConfiguration.java

```diff
@@ -1,15 +1,122 @@
 package com.crud.tasks.config;
 
+import com.crud.tasks.controller.TaskController;
+import io.swagger.annotations.Api;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.validation.MessageCodesResolver;
+import org.springframework.validation.Validator;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.HandlerMethodReturnValueHandler;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.config.annotation.*;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
+import java.lang.annotation.Annotation;
+import java.util.List;
+
+@EnableSwagger2
 @Configuration
-public class CoreConfiguration {
+public class CoreConfiguration implements WebMvcConfigurer {
 
     @Bean
     public RestTemplate restTemplate() {
         return new RestTemplate();
     }
 
+    @Bean
+    public Docket api() {
+
+
+        return new Docket(DocumentationType.SWAGGER_2)
+                .select()
+                .apis(RequestHandlerSelectors.withClassAnnotation(Api.class))
```

Comment:
Moim zdaniem to trochę coś innego. Dodając do pakietu kontrollery, musisz pamiętać o dodawaniu adnotacji @Api. To może z czasem prowadzić do "błędu" - dlaczego nie lapie mi, skoro dałem to w pakiecie z moimi kontrolerami...
`.apis(RequestHandlerSelectors.basePackage("com.crud.tasks.controller"))`